### DISCUSSION TITLE
GPT-J extra (pad) token

### DISCUSSION THREAD

**Discussion by attafosu** [mlcommons/inference#1533] [General]
During the finetuning of the GPT-J model, an extra token (pad="[PAD]") was inadvertently introduced as part of the model, which increased the `lm_head` dimension by 1 (from 50400 to 50401).

This extra token is now practically redundant (both in dataset [preprocessing](https://github.com/mlcommons/inference/blob/master/language/gpt-j/dataset.py#L44) and [post-processing](https://github.com/mlcommons/inference/blob/master/language/gpt-j/backend.py#L63) if padding is used during batching).

Creating a thread to discuss making it optional to use 50401 vs 50400 in the `lm_head`.

**Comment by mrmhodak** [comment]
WG feedback is that this is okay. No code change is required, this is up to submitters.
